### DISCUSSION TITLE
Open document pdf in new Tab in title link

### DISCUSSION THREAD

**Discussion by apolorc** [microsoft-search/pnp-modern-search#3458] [Ideas]
**Version used**
Ex: 4.10

Hi guys!

Is there any form to get that link title for PDF document opens in a new tab?

Thanks in advanced,

**Comment by kasperbolarsen** [comment]
Which layout template are you using?

**Comment by kasperbolarsen** [comment]
Adding data-interception="off" to the HTML / Handlebar expression should enable that, please see:
https://github.com/microsoft-search/pnp-modern-search/discussions/2932

**Reply by wobba** [reply]
Also see official docs at https://learn.microsoft.com/en-us/sharepoint/dev/spfx/hyperlinking
